src/graph_gui.cpp

```diff
@@ -492,7 +492,14 @@ struct BaseGraphWindow : Window {
 		/* Initialise the dataset */
 		this->UpdateStatistics(true);
 
-		this->InitNested(number);
+		this->CreateNestedTree();
+
+		auto *wid = this->GetWidget<NWidgetCore>(WID_GRAPH_FOOTER);
+		if (wid != nullptr && TimerGameEconomy::UsingWallclockUnits()) {
+			wid->SetDataTip(STR_GRAPH_LAST_72_MINUTES_TIME_LABEL, STR_NULL);
```

Comment:
Should we, for consistency, also have a string when it's not using wall clock units?
This PR keeps it as is, but it feels weird only having a tooltip some of the times. Possibly not for this PR though.